arch/riscv64/riscv64.cfg

```diff
@@ -1,5 +1,5 @@
 # Doesn't yet work right on RISC-V
-# CONFIG_DYNAMIC_FTRACE is not set
+CONFIG_DYNAMIC_FTRACE=n
```

Comment:
While it makes sense, I would prefer this change in particular to be separated in another PR, just so we can perform just this small change in meta-lmp and validate if the kernel config is indeed the same for all targets.

Reply:
Done